torch/_tensor_str.py

```diff
@@ -42,6 +44,7 @@ def set_printoptions(
             None (default) is specified, the value is defined by
             `torch._tensor_str._Formatter`. This value is automatically chosen
             by the framework.
+        show_shape: Print shape of the tensor. (default = True)
```

Comment:
default = False

Reply:
More correctly this would be "Whether to print the shape of strided, non-meta tensors. (default = False)"

Reply:
Done :heavy_check_mark: